Week1/README.md

```diff
@@ -16,12 +16,15 @@ These are the topics for week 1:
    - Manipulating elements
    - Browser events
    - Event listeners and handlers
+4. [UI Testing](https://study.hackyourfuture.net/#/testing/ui-tests.md)
 
 ## Week goals
 All of the detailed information about every concept is in your study book. The following goal explanation will link you to the correct place to get studying. Read through the goal description once to get an idea of what you will be learning, then go through the different links (either from the agenda or this goal list).
 
 This week we will learn about how we can use JavaScript to react to user interactions and change the website on the fly. We'll start off by learning about browsers [here](https://study.hackyourfuture.net/#/the-internet/browser). After that we will look into something called the DOM that forms the framework of how a website is built, read all about that [here](https://study.hackyourfuture.net/#/the-internet/dom). Then the fun part starts and we will have a look at how all the things we learned in the previous module can be used to actually interact with this DOM and adjust to pretty much anything the user does. Have a look [here](https://study.hackyourfuture.net/#/the-internet/dom-manipulation).
 
+While working with the DOM we need to also keep thinking about how we can test what we build. Testing using the DOM gets a lot more complicated as we need to somehow interact with the browser during our tests which is not easy. There are ways to do it, but later on in the curriculum we will be going into frontend frameworks that will actually take a lot of the difficulty away. This means for now that we will not be testing our code (if you are interested, your homework is being tested using puppeteer, have a look at those tests). We do want to introduce the concept of UI testing however, especially the section on writing testable UI's as we will be asking you to do that for the group project. Have a look [here](https://study.hackyourfuture.net/#/testing/ui-tests.md)!
+
```

Comment:
Actually, the unit tests use JSDOM rather than Puppeteer. The latter was found to not yet run natively on Apple silicon. This may have changed since, though.

Reply:
Right, forgot that was changed :D